docs/generateDocs.ps1

```diff
@@ -101,12 +101,20 @@ Write-Host "Repo root $repo_root, build root $build_root"
 # Clear output dir
 if( $incremental -eq $false ) {
     Write-Host "Clear previous version from $build_root"
-    Remove-Item -Force -Recurse $build_root
+    Remove-Item -Force -Recurse $build_root -ErrorAction Ignore
 }
 
 mkdir -ErrorAction Ignore $build_root | out-null
 mkdir -ErrorAction Ignore $cache_root | out-null
 
+if ((Get-ChildItem $build_root | Measure-Object).Count -ne 0)
```

Comment:
lol - our error checking is getting convoluted. "ErrorAction Ignore" is beginning to look like a code smell.
What should we do? (even if we don't do it now). Perhaps:

```
mkdir -force $build_root # note no erroraction: -force already ignores existing
if( $incremental == false ) {
  # Remove $build_root\* recursively
}```